src/controllers/command_controller.js

```diff
@@ -0,0 +1,185 @@
+/**
+ * Command Controller.
+ *
+ * Manages HTTP requests to /command.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+'use strict';
+
+const express = require('express');
+const Constants = require('../constants.js');
+const rp = require('request-promise');
+
+const CommandController = express.Router();
+
+
+//TODO:  move the Auth key somewhere else!
```

Comment:
Yeah this definitely can't be hard coded.

Reply:
Yes, maybe we can add the key here: https://github.com/mozilla-iot/gateway/blob/master/config/default.js